account/accountdomain/user/user.go

```diff
@@ -16,8 +16,10 @@ var (
 type User struct {
 	id            ID
 	name          string
-	displayName   string
+	alias         string
+	description   string
 	email         string
+	website       string
```

Comment:
let's create a seperate field named metadata that will have personal info instead of the application related info. i.e description, website can be part of that field.